measurements/templates/files/list.html

```diff
@@ -126,7 +126,7 @@
         {{ report_file.test_name }}
       </td>
       <td>
-        <a href="/files/download/{{ report_file.textname.split('/')[-1] }}">Download</a>
```

Comment:
I do not have enough context to understand why here we don't split anymore.

Reply:
This is actually a bug that was fixed. Since `textname` already includes the `bucket` inside of it's path, we don't need to just take the last part, but we need to specify the full path inclusive of the bucket.

Reply:
okay!